snapcraft/hooks/configure

```diff
@@ -5,42 +5,9 @@ if [ -d /sys/kernel/security/apparmor ] && [ "$(cat /proc/self/attr/current)" !=
     exec aa-exec -p unconfined -- "$0" "$@"
 fi
 
-# Utility functions
-get_bool() {
-    value=$(echo "${1:-}" | tr '[:upper:]' '[:lower:]')
-
-    # See if it's true
-    for yes in "true" "1" "yes" "on"; do
-        if [ "${value}" = "${yes}" ]; then
-            echo "true"
-            return
-        fi
-    done
-
-    # See if it's false
-    for no in "false" "0" "no" "off"; do
-        if [ "${value}" = "${no}" ]; then
-            echo "false"
-            return
-        fi
-    done
-
-    # Invalid value (or not set)
-    return
-}
-
-verify_int () {
-    value="${1:-}"
-
-    # Verify if the value is a positive integer
-    if echo "${value}" | grep -Eq '^[0-9]+$'; then
-        echo "${value}"
-        return
-    fi
-
-    # Invalid value (or not set)
-    return
-}
+# shellcheck disable=SC2155
+export SNAP_CURRENT="$(realpath "${SNAP}/..")/current"
```

Comment:
Can we assign and export on separate lines to avoid the need to disable the shellcheck ala https://www.shellcheck.net/wiki/SC2155